package.json

```diff
@@ -12,15 +12,16 @@
     "package": "ncc build src/index.ts --source-map --license licenses.txt",
     "version": "yarn build && yarn package && git add -f -- dist/index.js",
     "fixture:pack": "rm -f fixture.tgz && tar czf fixture.tgz test-fixture",
-    "fixture:setup": "rm -rf test-fixture && tar xzf fixture.tgz && yarn jest -c test-fixture/jest.config.js --coverage",
+    "fixture:setup": "rm -rf test-fixture && tar xzf fixture.tgz && yarn --cwd test-fixture jest -c test-fixture/jest.config.js --coverage",
```

Comment:
Jest generates the lcov coverage with relative file paths to the project root. Would want to have them relative to the `test-fixture` to match reality. Changing the cwd like so doesn't help. Any other ideas how to do this?